substrate/frame/support/test/tests/split_ui.rs

```diff
@@ -32,5 +32,5 @@ fn split_ui() {
 
 	let t = trybuild::TestCases::new();
 	t.compile_fail("tests/split_ui/*.rs");
-	t.pass("tests/split_ui/pass/*.rs");
+	t.pass("tests/split_ui/pass/split_*.rs");
```

Comment:
another solution is to put the not compiling module at `pass/storage/mod.rs`.
But both solution are ok to me.

Reply:
That's better.

The issue is a common mistake I guess and the tests somehow miss this case.